src/node_modules/app/machines/builder/form.options.js

```diff
@@ -181,10 +267,10 @@ function computeRequestInternal (
 		selection.field,
 		fieldInfo
 	);
-	const computedParams = {
+	const computedParams = JSON.parse(JSON.stringify({ // FIXME need deep copy
```

Comment:
I'd prefer to use `just-clone`, although I think we won't need this with the other changes I mentioned elsewhere to avoid modifying things in-place.